app/services/sufia/admin_set_create_service.rb

```diff
@@ -7,7 +7,7 @@ def self.create_default!
       return if AdminSet.exists?(DEFAULT_ID)
       admin_set = AdminSet.new(id: DEFAULT_ID, title: ['Default Admin Set'])
       begin
-        new(admin_set, nil).create
+        new(admin_set, nil, 'default').create
```

Comment:
I thought about that but I wasn't sure it would be possible to guarantee it was the first. Currently the .internal_test_app loads 2 workflows and I suspect it's a coincidence of alphabetization that 'default' is first and 'one_step_mediated_deposit' is second.